Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "filenamify"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Considering the MSRV might be updated in the next release of this crate, it will be a major(0.x) release rather than a patch(0.1.x) release. Therefore, you may just leave it unchanged in this PR. I will bump this separately when we get this PR merged.